cmake/build_configurations/compiler_options.cmake

```diff
@@ -43,7 +43,10 @@ IF(UNIX)
 
   # Default GCC flags
   IF(CMAKE_COMPILER_IS_GNUCC)
-    SET(COMMON_C_FLAGS "-fabi-version=2 -fno-omit-frame-pointer -fno-strict-aliasing")
+    SET(COMMON_C_FLAGS "-fno-omit-frame-pointer -fno-strict-aliasing")
+    IF(CMAKE_CXX_COMPILER_VERSION VERSION_LESS 10.0) # gcc-9 or older
+      STRING(APPEND COMMON_C_FLAGS " -fabi-version=2")
+    ENDIF()
```

Comment:
@altmannmarcelo removing fabi-version=2 for gcc > 10, doesn't seem to be right.  Afaiu, this is ABI check and should be available in newer versions too. Are we using c++ STL wrongly?

Reply:
@satya-bodapati for PXB 8.0 the exact same code doesn't produce the same issue. I checked and 8.0 doesn't enforce any `fabi-version`. Also, PS has the same fix https://github.com/percona/percona-server/blob/Percona-Server-5.7.36-39/cmake/build_configurations/compiler_options.cmake#L58

Reply:
ok, you beat me with sufficient proof :laughing: